frameworks/Python/bottle/setup.sh

```diff
@@ -1,7 +1,7 @@
 #!/bin/bash
-export PY2_ROOT=$IROOT/py2
-export PY2=$PY2_ROOT/bin/python
-export PY2_PIP=$PY2_ROOT/bin/pip
-export PY2_GUNICORN=$PY2_ROOT/bin/gunicorn
 
-$PY2_GUNICORN app:app -c gunicorn_conf.py &
\ No newline at end of file
+fw_depends python2
+
+$PY2_ROOT/bin/pip install --install-option="--prefix=${PY2_ROOT}" -r $TROOT/requirements.txt
+
+gunicorn app:app -c gunicorn_conf.py &
```

Comment:
Thanks! I'm super happy about 1) removing the need of `install.sh` and 2) improving the implementation of `fw_depends` to make it SUPER easy for contributors!